contracts/Comptroller.sol

```diff
@@ -199,6 +202,46 @@ contract Comptroller is
         return results;
     }
 
+    /**
+     * @notice Unlist an market by setting isListed to false
+     * @dev Pauses all actions, sets borrow/supply caps to 0 and sets collateral factor to 0.
+     * @param market The address of the market (token) to unlist
+     * @return uint256 Always NO_ERROR for compatibility with Venus core tooling
+     */
+    function unlistMarket(address market) external returns (uint256) {
+        _checkAccessAllowed("unlistMarket(address)");
+
+        Market storage _market = markets[market];
+
+        if (!_market.isListed) {
+            revert MarketNotListed(market);
+        }
+
+        _setActionPaused(market, Action.BORROW, true);
```

Comment:
Similar comment to https://github.com/VenusProtocol/venus-protocol/pull/429/files#r1500990555